pages/workflow.md

```diff
@@ -1,18 +1,31 @@
-# Workflow
+[Home](https://michaelrappa.github.io/usa-migration/) |
+[Analysis](https://michaelrappa.github.io/usa-migration/pages/analysis.html) |
+[Background](https://michaelrappa.github.io/usa-migration/pages/background.html) |
+[Reflections](https://michaelrappa.github.io/usa-migration/pages/reflections.html) |
+[Workflow](https://michaelrappa.github.io/usa-migration/pages/workflow.html) |
+[Credits](https://michaelrappa.github.io/usa-migration/pages/credits.html) |
+[Bibliography](https://michaelrappa.github.io/usa-migration/pages/bibliography.html) |
+[Sources](https://michaelrappa.github.io/usa-migration/pages/sources.html)
 
-Describe your overall workflow here. Think of this as something like when a scientific paper describes their process. The idea is *repeatability*. People need to know how they can get a copy of the data. If you chopped up the data, altered it, or joined it with other data sets they need to know that too.
 
-Discuss (in detail) the processes of
 
-* Getting the data
-* Cleaning/altering the data
-* Visualizing the data
-* Discussing and sharing results with group members
+# Data Processing Workflow
 
-What you need to cover: Talk about every step in your process, talk about how and why you made the decisions that you did. Why did you pick the website that you did? Did you download every record, or just some? Did you download the data directly from the website, or did you use a tool like Webscraper.io? Did you use OpenRefine to change the raw data in some form, and if so, why did you change it the way you did? What program(s) or techniques did you decide on to visualize or analyze it? Why did you feel those programs (e.g. Tableau or Gephi) were better than other options, given your research interest? Did you have to do anything to the data inside Tableau?
+* First I started by getting the Data from the website metrocosm.com which had an excel sheet that had all of the immigration Data that we needed which was from 1825-2017.
+* I then started to alter the excel sheet by flipping it to make it work for Tableau to make the visualizations.
+* I then made three visualizations which I used area charts for all of them which helped show the results of our data clearer.
+* As soon as I finished making my visualizations the data showed the increases and decreases of migrations to the United States then I showed my group the visualizatiuons and we got to see the results.
```

Comment:
You don't need to say it like this, but following the 'royal we', you could just say that as soon as the visualizations were finished, we saw the fluctuations in migrations to the United States.